From 326cd9606f2763be23cf35b3bdf46886f7110398 Mon Sep 17 00:00:00 2001
From: Badiuzzaman Iskhandar <badiuzzaman.azzarfan.bin.iskhandar@intel.com>
Date: Fri, 28 Dec 2018 03:09:23 +0800
Subject: [PATCH 0940/1214] drm/i915/huc: Update expected BXT HuC firmware to
 v01.8.2893

The huc firmwrae version should be aligned with firmware name
bxt_huc_ver01_8_2893.bin

Tracked-On: OAM-73872
Change-Id: Ie82d6ed9058907168a70ce88aefdcdb38a034f66
Signed-off-by: Badiuzzaman Iskhandar <badiuzzaman.azzarfan.bin.iskhandar@intel.com>
Signed-off-by: Hui Xia <hui.xia@intel.com>
---
 drivers/gpu/drm/i915/intel_huc_fw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_huc_fw.c b/drivers/gpu/drm/i915/intel_huc_fw.c
index 070ce12..9612227 100644
--- a/drivers/gpu/drm/i915/intel_huc_fw.c
+++ b/drivers/gpu/drm/i915/intel_huc_fw.c
@@ -23,7 +23,7 @@
  */
 
 #define BXT_HUC_FW_MAJOR 01
-#define BXT_HUC_FW_MINOR 0x08
+#define BXT_HUC_FW_MINOR 8
 #define BXT_BLD_NUM 2893
 
 #define SKL_HUC_FW_MAJOR 01
-- 
2.7.4

